bittensor/core/settings.py

```diff
@@ -46,19 +46,14 @@
 FINNEY_TEST_ENTRYPOINT = "wss://test.finney.opentensor.ai:443/"
 ARCHIVE_ENTRYPOINT = "wss://archive.chain.opentensor.ai:443/"
 LOCAL_ENTRYPOINT = os.getenv("BT_SUBTENSOR_CHAIN_ENDPOINT") or "ws://127.0.0.1:9944"
+SUBVORTEX_ENTRYPOINT = "ws://subvortex.info:9944"
 
 NETWORK_MAP = {
     NETWORKS[0]: FINNEY_ENTRYPOINT,
     NETWORKS[1]: FINNEY_TEST_ENTRYPOINT,
     NETWORKS[2]: ARCHIVE_ENTRYPOINT,
     NETWORKS[3]: LOCAL_ENTRYPOINT,
```

Comment:
```suggestion
NETWORK_MAP = {
    "finney": FINNEY_ENTRYPOINT,
    "test": FINNEY_TEST_ENTRYPOINT,
    "archive": ARCHIVE_ENTRYPOINT,
    "local": LOCAL_ENTRYPOINT,
    "subvortex": SUBVORTEX_ENTRYPOINT,
}
```
and later
```
NETWORKS = NETWORK_MAP.keys()
```
Normally I'd refactor the module to not use NETWORKS at all, but someone might be using it in subnet code, so lets leave it. One day when the interface is documented we can clean it up.